### ISSUE TITLE
No 1.23.0 release for VisualStudio

### ISSUE THREAD

**Issue by fdw** [Seddryck/NBi#675]
Hi, I just noticed that there doesn't seem to be a 1.230.0 release for `NBi.VisualStudio`. The last one (1.22.1) still depends on `NBi.Framework` v1.22.0.
Is that done on purpose?

**Comment by Seddryck** [comment]
> Is that done on purpose?

No, I just forgot to pack it and upload it (not part of the CI/CD).

**Comment by fdw** [comment]
Would it be possible to release it, please? 🙂
